R/produce_source_extract_tests.R

```diff
@@ -20,7 +22,12 @@
 #' @seealso [create_hb_test_flags()],
 #' [create_hscp_test_flags()] and [create_hb_cost_test_flags()]
 #' for creating test flags
-produce_source_extract_tests <- function(data) {
+produce_source_extract_tests <- function(data,
+                                         all_vars = c("beddays", "cost", "yearstay"),
```

Comment:
perhaps change to something more informative? is all sum and mean? maybe

```suggestion
                                         sum_mean_vars = c("beddays", "cost", "yearstay"),
```